Directory.Packages.props

```diff
@@ -55,16 +77,20 @@
     <PackageVersion Include="Serilog.Sinks.Async" Version="1.5.0" />
     <PackageVersion Include="Serilog.Sinks.File" Version="5.0.0" />
     <PackageVersion Include="Serilog.Sinks.Map" Version="1.0.2" />
-    <PackageVersion Include="SixLabors.ImageSharp" Version="3.0.2" />
+    <PackageVersion Include="SixLabors.ImageSharp" Version="3.1.0" />
```

Comment:
SixLabors.ImageSharp has been updated from 3.0.2 to 3.1.0: https://github.com/SixLabors/ImageSharp/releases/tag/v3.1.0

Reply:
And reverted, as 3.1.0 contains breaking changes and therefore isn't compatible/breaks ImageSharp.Web 3.0.x 😢